politeiawww/api/v1/v1.go

```diff
@@ -540,6 +545,25 @@ type VerifyUserPaymentReply struct {
 	PaywallTxNotBefore int64  `json:"paywalltxnotbefore"` // Minimum timestamp for paywall tx
 }
 
+// Users is used to request a list of users given a filter.
+type Users struct {
+	Username string `json:"username"`
+	Email    string `json:"email"`
+}
+
+// UsersReply is a reply to the Users command, replying with a list of users.
+type UsersReply struct {
+	Total uint64         `json:"total"`
```

Comment:
Comments documenting the struct fields are needed.  I wasn't sure if the was the total number of matches found or  the total number of users in the db.

Is the total number of users in the db something that's required?